Dockerfile.test

```diff
@@ -1,19 +1,21 @@
-FROM alpine:3.3
+FROM ubuntu:14.04
```

Comment:
this image is 100x larger, so this will take longer. Can we use 16.04? @ricarkol